.gitignore

```diff
@@ -9,6 +9,7 @@
 .docusaurus
 .cache-loader
 DIPMetadata.json
+.idea
```

Comment:
I think the intent here is that this should be in your local ignore and not the shared ignore for this repo. I don't use idea, I use vim, so if everyone included special files for their favorite editor, this would become unmanageable and unclear what is doing what.